pkg/config/config.go

```diff
@@ -538,17 +599,11 @@ type VersionTransform struct {
 
 // Schedule defines the schedule for the update check to run
 type Schedule struct {
-	Daily   bool `json:"daily,omitempty" yaml:"daily,omitempty"`
-	Weekly  bool `json:"weekly,omitempty" yaml:"weekly,omitempty"`
-	Monthly bool `json:"monthly,omitempty" yaml:"monthly,omitempty"`
-}
-
-// Polling indicates that polling should be used to find latest versions
-type Polling struct {
-	// Git should be used to find the latest version
-	Git bool `json:"git,omitempty" yaml:"git,omitempty"`
-	// The reason polling is being used
-	Reason string `json:"reason,omitempty" yaml:"reason,omitempty"`
+	// The reason scheduling is being used
+	Reason  string `json:"reason,omitempty" yaml:"reason,omitempty"`
+	Daily   bool   `json:"daily,omitempty" yaml:"daily,omitempty"`
+	Weekly  bool   `json:"weekly,omitempty" yaml:"weekly,omitempty"`
+	Monthly bool   `json:"monthly,omitempty" yaml:"monthly,omitempty"`
```

Comment:
Should this be something like `period: daily|weekly|monthly` (a string enum thing)?

Will something ever be both `daily: true` and `monthly: true` ? 🤔

Reply:
yeah sounds good to me!